src/app/problems/services/problems.service.ts

```diff
@@ -1,20 +1,33 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
-import { map } from 'rxjs/operators';
+import { map, tap} from 'rxjs/operators';
 import { Problem } from '../models/problem.model';
 import { AppConfig } from '../../app.config';
 import { HttpClient } from '@angular/common/http';
+import { normalize } from 'normalizr';
+import { problems } from '../../core/schemas/data.schema';
 
 @Injectable()
 export class ProblemsService {
 
     constructor(private http: HttpClient, private config: AppConfig) {
     }
 
-    get(): Observable<Problem[]> {
+    get(): Observable<any> {
```

Comment:
Why any? It could be problem with problem references right?

Reply:
It's the normalizer nature. The normalize() function returns object like:
{
entities: any,
result: any
}

We could create an auxiliary ProblemData model that could be:
{
entities: Problem[],
result: any
}

and we could do get(): Observable<ProblemData> . That's doable but having it like that is not a risk in this place. But we can keep it in mind for some refactor stories